pkgs/by-name/tt/ttaenc/package.nix

```diff
@@ -0,0 +1,44 @@
+{
+  stdenv,
+  lib,
+  fetchurl,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  name = "ttaenc";
+  version = "3.4.1";
+
+  src = fetchurl {
+    url = "http://downloads.sourceforge.net/tta/${finalAttrs.name}-${finalAttrs.version}-src.tgz";
+    sha256 = "sha256-ssnIsBWsxYZPCCoBV/LgnFEX0URTIctheOkltEi+PcY=";
+  };
+
+  patches = [
+    ./makefile.patch # Use stdenv's CC
+    ./ttaenc-inline.patch # Patch __inline used into always_inline for both GCC and clang
+  ];
+
+  makeFlags = [ "INSDIR=$(out)/bin" ];
+
+  preBuild = ''
+    # From the Makefile, with `-msse` removed, since we have those on by x86_64 by default.
+    makeFlagsArray+=(CFLAGS="-Wall -O3 -fomit-frame-pointer -funroll-loops -fforce-addr -falign-functions=4")
```

Comment:
Unless O3 is significantly faster we usually stick to O2

Reply:
These flags are straight from the `Makefile` of the project, so I don't really want to diverge from it. I only removed `-msse` since it's not a flag on aarch64 gcc

Reply:
In that case you could remove `-Wall` (not useful for us), but it's also fine as is